src/components/Modals/Send/utils.ts

```diff
@@ -186,15 +196,32 @@ export const handleSend = async ({
       if (fromChainId(sendInput.asset.chainId).chainNamespace === CHAIN_NAMESPACE.CosmosSdk) {
         const fees = estimatedFees[feeType] as FeeData<CosmosSdkChainId>
         const { accountNumber } = bip44Params
-        return adapter.buildSendTransaction({
+        const params = {
           to,
           memo: (sendInput as SendInput<CosmosSdkChainId>).memo,
           value,
           wallet,
           accountNumber,
           chainSpecific: { gas: fees.chainSpecific.gasLimit, fee: fees.txFee },
           sendMax: sendInput.sendMax,
-        })
+        }
+        const { assetReference, assetNamespace } = fromAssetId(sendInput.asset.assetId)
+        if (
+          sendInput.asset.chainId === KnownChainIds.OsmosisMainnet &&
+          assetNamespace === ASSET_NAMESPACE.ibc
+        ) {
+          return (
+            adapter as unknown as CosmosSdkBaseAdapter<KnownChainIds.OsmosisMainnet>
+          ).buildSendTransaction({
+            ...params,
+            chainSpecific: {
+              ...params.chainSpecific,
```

Comment:
Can use `lodash/merge` or immer to avoid the deep spread, fine here but a good practice to avoid spread madness